### PR TITLE
fixed a typo

### PR DESCRIPTION
**Pull request by jurajsucik**

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/recipients-in-exchange-online/plus-addressing-in-exchange-online.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ You can create new plus addresses by adding a new tag. You can use plus addresse
 >
 > Some web forms don't support plus signs in email addresses.
 >
-> If you need to unsubscribe from an email list subscription service, some subscription services require that use the original email address that you subsribed with. You can't unsubscribe by sending email from plus address.
+> If you need to unsubscribe from an email list subscription service, some subscription services require that use the original email address that you subscribed with. You can't unsubscribe by sending email from plus address.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/recipients-in-exchange-online/plus-addressing-in-exchange-online.md:55]:
```suggestion
> If you need to unsubscribe from an email list subscription service, some subscription services require that you use the original email address that you subscribed with. You can't unsubscribe by sending an email from a plus address.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @jurajsucik, kindly commit the suggested changes. Many thanks.

File: Exchange/ExchangeOnline/recipients-in-exchange-online/plus-addressing-in-exchange-online.md (hunk 1 above)
Comment (jurajsucik) [reply Exchange/ExchangeOnline/recipients-in-exchange-online/plus-addressing-in-exchange-online.md]: suggestion committed

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by jurajsucik** [comment]
can this PR be merged now?

**Pull request merged**